tests/test_pysoundfile.py

```diff
@@ -371,7 +371,7 @@ def test_blocks_with_frames_and_fill_value(file_stereo_r):
 def test_blocks_with_out(file_stereo_r):
     out = np.empty((3, 2))
     blocks = list(sf.blocks(file_stereo_r, out=out))
-    assert blocks[0] is out
+    assert blocks[0].base is out
```

Comment:
Why does this need to be changed?

Reply:
I used slightly inefficient slicing. This should be back to the original assertion now.